python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -48,7 +48,7 @@ def json_to_vpdq(json_str: str) -> t.List[vpdq.VpdqFeature]:
     for frame_number, feature in vpdq_json.items():
         features.append(
             vpdq.VpdqFeature(
-                feature[QUALITY], frame_number, feature[HASH], feature[TIMESTAMP]
+                feature[QUALITY], int(frame_number), feature[HASH], feature[TIMESTAMP]
```

Comment:
It is because json object key is automatically converted to string (javascript only accepts string as key). That's why a cast is needed when loading the jsons with frame_number as the key.